### ISSUE TITLE
when project version is a SNAPSHOT, `rpm:packageBin` fails with "`version in Rpm` is empty."

### ISSUE THREAD

**Issue by colin-passiv** [sbt/sbt-native-packager#1004]
## Expected behaviour

When version is a snapshot, and version in Rpm has been explicitly set (to a value without a -SNAPSHOT suffix), e.g
build.sbt
```
version := "0.0.1-SNAPSHOT"
version in Rpm := "1"
```
, the resulting rpm should have the version as set in `version in Rpm`.

## Actual behaviour

Build fails with error
```
`version in Rpm` is empty.  Please provide a vaid version for the rpm SPEC.
java.lang.RuntimeException: There are issues with the rpm spec data.
```

## Information

- What sbt-native-packager are you using
1.2.0
(worked in 1.2.0-M9)
- What sbt version
0.13.15
- What is your build system (e.g. Ubuntu, MacOS, Windows, Debian )
CentOS 6
- What package are you building (e.g. docker, rpm, ...)
rpm
- What version has your build tool (find out with e.g. `rpm --version`)
RPM version 4.8.0
- What is your target system (e.g. Ubuntu 16.04, CentOS 7)
CentOS 6

**Comment by vpavkin** [comment]
I got the same on 1.2.0-M9 after switching from scala 2.11 to scala 2.12.

**Comment by vpavkin** [comment]
hm, seems like sbt unconditionnaly resolves 1.2.0 for me, even when I specify milestone version.

**Comment by muuki88** [comment]
Fixed in mentioned PR (#1005)
